### ISSUE TITLE
play.geforcenow.com ads

### ISSUE THREAD

**Issue by testtest221** [uBlockOrigin/uAssets#22887]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Search [here](https://github.com/uBlockOrigin/uAssets/issues) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL(s) where the issue occurs.

```yaml
play.geforcenow.com
```

### Description

Geforce NOW (a streaming service) recently introduced ads to its free tier, while waiting in the queue for a free rig you will get a video Ad that covers half of the screen and the ad will keep playing until your turn in the queue is up (although the ad doesn't always show up, only once every 3-5 sessions)

### Other extensions used

None

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![Screenshot_1](https://github.com/uBlockOrigin/uAssets/assets/163224301/aa72ca98-5251-451e-a542-11f44be96b52)

</details>

### Configuration

<details>
<summary>Details</summary>

```yaml
uBlock Origin: 1.56.0
Chromium: 122
filterset (summary):
 network: 117090
 cosmetic: 45601
 scriptlet: 20295
 html: 0
listset (total-discarded, last-updated):
 removed:
  urlhaus-1: null
 default:
  user-filters: 0-0, never
  easylist: 82291-15, 30m Δ
  easyprivacy: 50358-8, 30m Δ
  plowe-0: 3772-1184, 1h.30m
  ublock-badware: 7858-147, 30m Δ
  ublock-filters: 37463-355, 30m Δ
  ublock-privacy: 755-6, 30m Δ
  ublock-quick-fixes: 150-4, 30m Δ
  ublock-unbreak: 2244-35, 30m Δ
filterset (user): [empty]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 624 ms (selfie)
 maxAssetCacheWait: 266 ms
 cacheBackend: browser.storage.local

```
</details>

**Comment by stephenhawk8054** [comment]
I don't see any extensions (including uBO) running in your screenshot

**Comment by testtest221** [comment]
> I don't see any extensions (including uBO) running in your screenshot

Yes I didn't have uBlock running while taking that particular screenshot, but I did 5 or 6 other attemps with uBlock enabled and still got the Ad, however here is a picture with uBlock enabled:
![Screenshot_1](https://github.com/uBlockOrigin/uAssets/assets/163224301/79ce3de9-2c60-419c-a27b-1ca170bec5f0)
